### ISSUE TITLE
Request: Percent on ##/### Ranking

### ISSUE THREAD

**Issue by lbell** [thomaschampagne/stravistix#228]
Love the [23/6443] rank on the segments list on the activity page. If I click through to the full leaderboard, I can see my % ranking... But would love to have that info front and center on the segment list. Thanks!

[23/6443 - 1%]

 (I'm math challenged)

**Comment by thomaschampagne** [comment]
@lbell In the same square? Like this?

![image](https://cloud.githubusercontent.com/assets/151973/15274034/9660d212-1aa9-11e6-8d1a-0045e1d99980.png)

**Comment by thomaschampagne** [comment]
It's done. Ready for next release ;)

**Comment by lbell** [comment]
Beauty! Thanks

**Comment by aiacciu87** [comment]
Hello,

Sorry to bring up a post so old, but for a few days, the ranks in the list of segments (as in the screenshot above), no longer displayed.

Is this a known bug? I disabled all of my other Chrome extensions to see if there was a conflict. I uninstalled and re-installed Elevate App, nothing helps.

An idea ? Thanks!

![Image 2020-08-18 at 11 12 33 AM](https://user-images.githubusercontent.com/874950/90494384-bf7e3800-e143-11ea-9e19-1393df119458.png)
